api/grpcserver.go

```diff
@@ -671,7 +672,7 @@ func (svr *gRPCHandler) TraceTransactionStructLogs(ctx context.Context, in *iote
 	if !ok {
 		return nil, status.Error(codes.InvalidArgument, "the type of action is not supported")
 	}
-	tracer := vm.NewStructLogger(nil)
+	tracer := logger.NewStructLogger(nil)
```

Comment:
do we have to make this change in this PR?

Reply:
yes, due to EVM upgrade